test/Mono.Linker.Tests/TestCasesRunner/TestRunner.cs

```diff
@@ -109,7 +109,7 @@ private LinkedTestCaseResult Link (TestCase testCase, TestCaseSandbox sandbox, M
 			AddLinkOptions (sandbox, compilationResult, builder, metadataProvider);
 
 			LinkerTestLogger logger = new LinkerTestLogger ();
-			linker.Link (builder.ToArgs (), linkerCustomizations, logger);
+			linker.Link (builder.ToArgs (), linkerCustomizations, logger, testCase.Name == "CustomStepWithWarnings");
```

Comment:
The test "InvalidWarningCodeThrows" expects an ArgumentException to be thrown and fails if the process crashes. This flag makes sure we don't fail that test.